index.html

```diff
@@ -1,220 +1,275 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="./style.css" type="text/css">
-    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
-    <title>portfolio</title>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio Project</title>
+    <link rel="stylesheet" href="style.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;700&display=swap" rel="stylesheet"/>
+    <script src="https://kit.fontawesome.com/25935046d9.js" crossorigin="anonymous"></script>
   </head>
+
   <body>
```

Comment:
- [ ] Kindly fix image links: currently all images in your projects are not visible reason could be the links path changed, here is a video I recorded, please fix this ⬇️ 


https://user-images.githubusercontent.com/69638013/161382415-1109abc0-9417-42ea-993f-0e4b57eb5d63.mp4